app.json

```diff
@@ -130,6 +130,10 @@
       "description": "Chunk size to use for Elasticsearch indexing tasks",
       "required": false
     },
+    "ELASTICSEARCH_DOCUMENT_INDEXING_CHUNK_SIZE": {
```

Comment:
Any reason not to reuse `ELASTICSEARCH_INDEXING_CHUNK_SIZE` here? Is the intention to have a smaller chunk size for courses than for other indexed items?